resources/infrastructure_logger.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <logging>
+    <testMode filePath="resources/infrastructure_logger.xml"></testMode>
```

Comment:
The path is pointing is to same file. Why not have another xml called testing_logger.xml(under resources) with the format for testing? Also a boolean flag for test mode might be better than rewriting the path every time

Reply:
yeah that's what I told in the PR comment:
>If the path is to be changed, let me know
I'll create a file named testing_logger.xml then